packages/core/EventManager.ts

```diff
@@ -376,32 +377,37 @@ export default class EventManager {
       // As the reschedule requires confirmation, we can't update the events and meetings to new time yet. So, just delete them and let it be handled when organizer confirms the booking.
       await this.deleteEventsAndMeetings({ booking, event });
     } else {
-      // If the reschedule doesn't require confirmation, we can "update" the events and meetings to new time.
-      const isDedicated = evt.location ? isDedicatedIntegration(evt.location) : null;
-      // If and only if event type is a dedicated meeting, update the dedicated video meeting.
-      if (isDedicated) {
-        const result = await this.updateVideoEvent(evt, booking);
-        const [updatedEvent] = Array.isArray(result.updatedEvent)
-          ? result.updatedEvent
-          : [result.updatedEvent];
-
-        if (updatedEvent) {
-          evt.videoCallData = updatedEvent;
-          evt.location = updatedEvent.url;
+      if (changedOrganizer) {
```

Comment:
I'm not sure how I feel about calling the event manager reschedule and the create methods back to back. How do we feel about calling only one event manager method and handling the deletion and creation of the calendar events?

Reply:
Agreed. IMO reschedule should handle event recreation. But could be done in a follow up